third_party/proton/proton/viewer.py

```diff
@@ -146,7 +143,12 @@ def get_time_seconds(df):
             derived_metrics.append(f"{metric} (inc)")
         else:
             original_metrics.append(metric)
-
+        if metric not in exclusive_metrics:
+             kernel = gf.dataframe[metric_name]
```

Comment:
nit: it's probably not a "kernel" since we also have individual lines representing as frames

Reply:
Oh, right.